gui/wxpython/core/watchdog.py

```diff
@@ -0,0 +1,219 @@
+"""
+@package core.watchdog
```

Comment:
Moving it to separate file is a good idea. Could you please clarify which parts of the file were moved but not actually modified? That would help with the review.

Reply:
Code is same as your original code. `MapsetWatchdog` class is wrapper around your original code.